### DISCUSSION TITLE
[Project board] Ability to sort by issue creation date

### DISCUSSION THREAD

**Discussion by NatalliaAlieva** [community/community#45953] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Hello. 

My question is about Project Board view. 
Do you plan to add sorting in the columns by issue creating date or by serial number? Or perhaps you can suggest any workaround for this?
A similar question was asked about the beta version more than a year ago, but there is still no answer. 

Our project really lacks this ability in the columns on the Project board.

**Comment by rileybroughten** [comment, marked as answer]
Thank you for this feedback @NatalliaAlieva. There is a separate [discussion](https://github.com/community/community/discussions/8518) with feedback and updates for this, I would suggest following along there for developments as we look to support this on Projects!

**Reply by NatalliaAlieva** [reply]
Thank you for your answer. 
I am following that discussion too. Hopefully this sorting option will be available soon.
Thank you for your work! :)

**Reply by rileybroughten** [reply]
Thank you for the feedback and patience as we work to support this!

**Reply by adrael** [reply]
Hey there @rileybroughten 👋 

Landing here from a quick google search, 1 year later, and it seems the feature is still not shipped — I need it for some grant prep in order to be able to accurately log issues in the proper timeframe.

Do you have visibility over this feature's GTM?
Much appreciated 🙏

**Reply by dliakin** [reply]
I came from google search too. Would like to have this option. I am using filters now as workaround

**Reply by sky-gualter-semedo** [reply]
+1

**Reply by stherold** [reply]
> I came from google search too. Would like to have this option. I am using filters now as workaround

@dliakin can you share how you do it?

**Reply by shmuel44** [reply]
+1

**Reply by BenZionKatz** [reply]
+1

**Comment by samus-aran** [comment]
Hey folks. We will be locking this discussion as @rileybroughten has already answered that there is a more active discussion of a similar request. 👍
